### ISSUE TITLE
Problem with selling

### ISSUE THREAD

**Issue by ChiefVitor** [fatbeard2/FUT17-autobuyer#6]
it seems that there is a problem when listing the players the buyer says it will list them up but it doesnt it only buys them and leaves them in the trade pile.
With Renato Sanches that happens.

**Comment by fatbeard2** [comment]
It works for me. Maybe its the same problem with wrong url. It just makes request for a wrong platform?

**Comment by ChiefVitor** [comment]
I changed everything to s2, maybe the cookie ?

**Comment by ChiefVitor** [comment]
With cheap players works but with more expensive nope.

**Comment by ChiefVitor** [comment]
req.body = {startingBid: ARGV[2].to_i - 100,

You need to change that to 250 to work ;) working fine now.

**Comment by ChiefVitor** [comment]
off 
Bruh do you know any good players to trade on monday?

**Comment by fatbeard2** [comment]
nope. I tried oscar, but didn't snipe a single card for a whole day :)

**Comment by ChiefVitor** [comment]
Can you tell me your skype?

**Comment by fatbeard2** [comment]
I've added you
